assets/src/blocks/Covers/CoversScript.js

```diff
@@ -1,7 +1,7 @@
-import { CoversFrontend } from './CoversFrontend';
+import {CoversFrontend} from './CoversFrontend';
 
-document.querySelectorAll( `[data-render='planet4-blocks/covers']` ).forEach(
-  blockNode => {
+document.querySelectorAll( '[data-render=\'planet4-blocks/covers\']' ).forEach(
```

Comment:
Is it invalid to change to
`document.querySelectorAll( '[data-render="planet4-blocks/covers"]' )` or `document.querySelectorAll( "[data-render='planet4-blocks/covers']" )`. I always disagree to `escape` chars when it can be avoided 🤭 .

Reply:
Thank you, fixed.